sound/soc/sof/loader.c

```diff
@@ -12,7 +12,7 @@
 
 #include <linux/firmware.h>
 #include <sound/sof.h>
-#include <uapi/sound/sof/ext_manifest.h>
+#include <sound/sof/ext_manifest.h>
```

Comment:
I also don't get this, I see the ext_manifest.h in include/uapi/sound/sof

You can't change the include path without changing the location of the file as well.

Reply:
So you are willing to keep original version - `#include <uapi/sound/sof/ext_manifest.h>` - or to create linking header `sound/sof/ext_manifest.h` ?

Reply:
@plbossart I *think* `include/uapi` is also among include paths for the kernel build system. Otherwise this wouldn't build. And I see "build completed" from CI below.

Reply:
I'd vote for "#include <uapi/sound/sof/ext_manifest.h>" Uapi is in the include search path, so both variants will compile, but this makes it explicit that the uapi header is used. Grepping trhough drivers/, i see 149 uses if "#include <uapi/foo/bar/something.h>" and we do same in soc-topology.c, so there is existing practise for this.